test/lib/absinthe/execution/arguments_test.exs

```diff
@@ -47,7 +47,18 @@ defmodule Absinthe.Execution.ArgumentsTest do
       value :sms, deprecate: "Use phone instead"
     end
 
```

Comment:
I moved the tests to arguments_test because that's generally where we've been doing these kinds of tests.